docs/UserGuide.md

```diff
@@ -159,4 +159,49 @@ Format: `delete INDEX`
 Examples:
 * `list` followed by `delete 3` deletes the 3rd Todo in the displayed Todos list.
 
-## bryan your work here (general)
+## General
+
+### Switching between Tabs: `tab`
+
+Format: tab /TAB NAME
+* Switches between different tabs. Tab names are as follows:
+    * dashboard
+    * contacts
+    * schedule
+    * todos
+
+Examples:
+* `tab dashboard` while in the Contacts tab changes to the Dashboard tab
+* `tab contacts` in the Contacts tab just jumps to the top of the page (returns the tab to the original landing page view)
+
+### Adding context to commands
+
+Adds a prefix to the commands (if applicable)
```

Comment:
Actually I'm unsure if phrasing it this way will make it come off as if "if user does x, a prefix to the commands will be added". I think the adding of a prefix is a backend detail (what I mean is if the parser sees no prefix it will auto-prefix the name of the tab the command was executed in, before parsing the command), the users don't need to know that, so maybe all we need to specify is something like "user can run commands meant for a particular tab from a different tab, by simply adding the name of the tab the user means to run the command on as a prefix to the command itself", then in the example maybe make it less backend, give them examples of how to add a contact from the todo tab, or how to delete an event from the dashboard, and maybe what happens if they prefix /schedule while already on the schedule tab.

Reply:
I agree with this. The written text is more for developers to understand how it is implemented. Perhaps it is better to rename the topic as `Convenience commands` and follow up with the relevant examples?